packages/@okta/vuepress-site/docs/guides/build-sso-integration/create-your-app/saml2/create.md

```diff
@@ -1,73 +1,34 @@
 ### Create a SAML integration
 
-If you don't have an Okta developer account, begin by signing up for one at <https://developer.okta.com/signup/>.
-
-1. After you request the developer account and have received the initialization email, click the link to go to your developer org. Sign in as a user with administrative privileges.
-1. Navigate to the Admin Console in your Okta org by clicking **Admin**.
-  ![Admin Button](/img/oin/scim_end-user-ui.png "Admin Button")
-1. If you are in the Developer Console, click **Developer Console** and then **Classic UI** to switch over to the Admin Console in your Okta org.
-  ![Switch to Admin Console](/img/oin/scim_switch-ui.png "Switch to Admin UI")
-1. Click **Applications** > **Applications**.
-  ![Open Applications](/img/oin/scim_open-apps.png "Open Applications")
-1. Click **Add Application**.
-  ![Create Application](/img/oin/scim_create-app.png "Add App button")
-1. Click **Create New App** to start the Application Integration Wizard.
-  ![Create New Application](/img/oin/scim_create-app-new.png "Create App button")
-1. Choose **Web** as the platform for your app. Web is the only supported platform for SAML 2.0 apps in the OIN.
-1. Select **SAML 2.0** in the **Sign on method** section.
+7. Select **SAML 2.0** in the **Sign on method** section.
 1. Click **Create**.
-1. On the **General Settings** tab, enter a name for your app and optionally upload a logo. You can also hide the app from your end-user's dashboard and mobile app.
-1. On the **Configure SAML** tab, use the SAML information that you gathered in the <GuideLink link="../before-you-begin">first step</GuideLink> to configure the settings of your app in Okta. These steps are also detailed in the SAML App Wizard section of [Using the App Integration Wizard](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard) in the Okta product documentation.
-      1. Enter the Assertion Consumer Service (ACS) URL into the **Single sign on URL** field.
+1. On the **General Settings** tab, enter a name for your integration and optionally upload a logo. You can also hide the integration from your end-user's Okta dashboard or mobile app.
+1. On the **Configure SAML** tab, use the SAML information that you gathered in the <GuideLink link="../before-you-begin">first step</GuideLink> to configure the settings of your integration. These steps are also detailed in the SAML App Wizard section of [Using the App Integration Wizard](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard) in the Okta product documentation.
+      1. In the **Single sign on URL** field, enter the Assertion Consumer Service (ACS) URL.
```

Comment:
We can't change this except by altering the CSS for the whole site (according to @brettritter-okta ) - at this stage, I'm okay with just going with numbers for the main tasks and bullets for the subtasks. They're fairly self evident that you need to go in order, but it's not going to kill you if you don't because they all need to be filled in.